Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -668,3 +668,35 @@ def onNodeAdded(caller, event, calldata):
   slicer.mrmlScene.RemoveObserver(sceneObserverTag)
 
   return loadedNodeIDs
+
+def loadFromDICOMWeb(dicomWebEndpoint, studyUID, seriesUID="", accessToken=""):
+  from dicomweb_client.api import DICOMwebClient
+  client = DICOMwebClient(
+              url = dicomWebEndpoint,
+              headers = { "Authorization": "Bearer {}".format(accessToken) },
+            )
+
+  seriesList = client.search_for_series(study_instance_uid=studyUID)
+  seriesUIDs = []
+  if seriesUID != "":
+    seriesUIDs = [seriesUID]
+  else:
+    for series in seriesList:
+      currentSeriesUID = series['0020000E']['Value'][0]
+      seriesUIDs.append(currentSeriesUID)
+
+  fileNumber = 0
+  for currentSeriesUID in seriesUIDs:
+    instances = client.retrieve_series(
+      study_instance_uid=studyUID,
+      series_instance_uid=currentSeriesUID)
+
+    outputDirectory = (slicer.dicomDatabase.databaseDirectory +
+      "/" + studyUID + "/" + currentSeriesUID + "/")
+    if not os.access(outputDirectory, os.F_OK):
+      os.makedirs(outputDirectory)
+    for instance in instances:
+      filename = outputDirectory + str(fileNumber) + ".dcm"
+      instance.save_as(filename)
+      fileNumber += 1
+    importDicom(outputDirectory)
```

Comment:
it could be nice to return loaded node IDs

Reply:
This function doesn't load nodes into slicer, just downloads and imports into the database.
I've renamed the function to importFromDICOMWeb to reflect this, and return a list of loaded series UIDs.